department-of-veterans-affairs/abd-vro
# mv domain-rrd to a new repo

## What was the problem?


Associated tickets or Slack threads:
- #2184 

## How does this fix it?[^1]

Removes code after the PR to add this code to another repo

## How to test this PR
- no functional changes

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
